channels/jp.m3u

```diff
@@ -1,6 +1,8 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Japanese" tvg-logo="https://i.imgur.com/s6f0LgS.jpg" group-title="News",BBC News Japan
 https://bbc1.media.ylive.jp/53ccb8ca9cb44762b3f1f4aecc85b7ba/ap-northeast-1/5690807595001/profile_0/chunklist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Japanese" tvg-logo="https://i.imgur.com/p0hYPsF.png" group-title="News",FNN Prime
+https://bcovlive-a.akamaihd.net/6efedafb488048f7b166290505320053/ap-northeast-1/6048574195001/playlist.m3u8
```

Comment:
This link shows only a short screen recording and then the stream stops:
https://streamtest.in/logs/https-bcovlive-a-akamaihd-net-6efedafb488048f7b166290505320053-ap-northeast-1-6048574195001-playlist-m3u8

Reply:
and now there's some other channel
<img width="825" alt="Captura de pantalla 2020-05-06 a las 23 23 56" src="https://user-images.githubusercontent.com/7253922/81224649-b46a2680-8ff0-11ea-9a13-6ee23fd48b30.png">